temporalcli/commandsmd/commands.md

```diff
@@ -2521,19 +2521,23 @@ temporal workflow trace \
 
 Includes options set for [workflow reference](#options-set-for-workflow-reference).
 
-### temporal workflow update: Synchronously run a Workflow update handler
+### temporal workflow update: Start and wait for Updates
```

Comment:
IMO we should mark all of these commands as experimental similar to their SDK counterparts